samples/notebooks/powershell/Docs/Profile Support.ipynb

```diff
@@ -0,0 +1,133 @@
+    "\n",
+    "|`$PROFILE` Property Name|Description|\n",
+    "|:------------------------|:-----------|\n",
+    "|`$PROFILE.CurrentUserCurrentHost`|The current user for the current host|\n",
+    "|`$PROFILE.AllUsersCurrentHost`|All users for the current host|\n",
+    "|`$PROFILE.CurrentUserAllHosts`|The current user for all hosts|\n",
+    "|`$PROFILE.AllUsersAllHost`|All users for all hosts|\n",
+    "\n",
+    "> NOTE: A \"host\" here refers to a PowerShell Host application or a \"PSHost\". These are applications that host a PowerShell runtime. Examples of these include:\n",
+    "> * ConsoleHost (used by `pwsh.exe`/`pwsh`)\n",
+    "> * PowerShell Editor Services Host (used by the PowerShell extension for Visual Studio Code's PowerShell Integrated Console)\n",
+    "> * .NET Interactive Host (used by this experience in .NET Interactive)\n",
+    "\n",
+    "Check out [the PowerShell documentation](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_profiles?#the-profile-variable) on the `$PROFILE` object for more information.\n",
+    "\n",
+    "### Differences in .NET Interactive's Profile implementation\n",
+    "\n",
+    "With all that said, .NET Interactive doesn't perfectly align with PowerShell profiles for a few good reasons:\n",
+    "\n",
+    "#### There is no support for `$PROFILE.CurrentUserAllHosts` or `$PROFILE.AllUsersAllHosts`\n",
+    "\n",
+    "A lot of folks customize their PowerShell prompt in the `$PROFILE.CurrentUserAllHosts` or `$PROFILE.AllUsersAllHosts` profiles. .NET Interactive doesn't have a concept of a prompt. Because of this, .NET Interactive does not run your `$PROFILE.CurrentUserAllHosts` or `$PROFILE.AllUsersAllHosts` profiles. If you feel like this isn't the right behavior, let us know!\n",
+    "\n",
+    "#### `$PROFILE.AllUsersCurrentHost` isn't in a system-wide location\n",
+    "\n",
+    "Try running it:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "$PROFILE.AllUsersCurrentHost"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You'll notice that it exists in the location that .NET Interactive was installed to.\n",
+    "\n",
+    "This is typical for the `AllUsersCurrentHost` profile as it typically installs in the result of:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "$PSHOME"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "However, since .NET Interactive is a global tool and global tools are a user installation and not a system-wide installation, the `AllUsersCurrentHost` path is where it is today."
```

Comment:
It's a .NET Core tool, and doesn't necessarily have to be installed globally, e.g. we install it as a local tool in the VS Code extension so that the version doesn't conflict with the globally-installed one.

It just occurred to me to ask -- will these two install styles use the the same PowerShell profile?

Reply:
They will have the same `$PROFILE.CurrentUserCurrentHost` but their `$PROFILE.AllUsers.CurrentHost` will be different.

I don't think the latter will be used at all... most folks will just use `$PROFILE.CurrentUserCurrentHost` which is the default value of `$PROFILE`

Reply:
Do you want me call it just a tool instead of a global tool?

Reply:
changed the wording a bit